modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/dlc/DLCQueueTestCase.java

```diff
@@ -102,17 +128,22 @@ public void createDLC() throws AndesClientConfigurationException, NamingExceptio
         publisherClient.startClient();
 
         //Thread sleep until messages sent to DLC after breaching maximum number of retrying
-        AndesClientUtils.sleepForInterval(100000L);
+        AndesClientUtils.sleepForInterval(150000L);
```

Comment:
This value haven't used in any other place. Since messages will retry to send several times before sending it to DLC we have to wait until certain time. This wait time can change from test case to test case as number of DLC messages counted can be different.